DependencyInjection/GosWebSocketExtension.php

```diff
@@ -30,6 +30,18 @@ public function load(array $configs, ContainerBuilder $container)
         $configuration = new Configuration();
         $configs = $this->processConfiguration($configuration, $configs);
 
+        // Set the SecurityContext for Symfony <2.6
+        // Should go back to simple configuration after drop <2.6 support
+        if (interface_exists('Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface')) {
```

Comment:
@ProPheT777 see this comment - https://github.com/symfony/symfony-docs/issues/5879#issuecomment-155122336
previous version with ```interface_exist``` is better choice.
In specific case ```HttpKernel```  component version may be different than ```Security```